lib/stripEndings.js

```diff
@@ -10,7 +10,10 @@ const optionalEndingClass = getRegexClass( [ '(' ] )
 const brokenEndingClass = getRegexGroup( [ '()' ] )
 
 // All numbers in ASCII, Unicode
-const numbers = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 ].map( i => i.toString() )
+const numbers = [
+  0, 1, 2, 3, 4, 5, 6, 7, 8, 9,
+  '१', '२', '३', '४', '५', '६', '७', '८', '९', '०',
```

Comment:
Don't do this here, better to do it in L17 as something like `numbers.map(toUnicode).map(toHindi)`, instead of duplicating

Reply:
Just sharing my thoughts: I still think it's shorter and easier to understand typed out explicitly, these numbers don't change, and they're basically constants in the unicode spec

Reply:
Hmm. Maybe you're right. I still suggested using the `toHindi` function, because we know that's correct, and I cannot read Hindi (or indeed any other languages we may add), and so cannot personally proof them.